static/src/stylesheets/module/content-garnett/_article-immersive.scss

```diff
@@ -565,7 +565,7 @@
 }
 
 //overides the immersive article — done like this to keep gallery's and photo eassy in ok shape
-.content--immersive:not(.immersive-main-media__gallery):not(.paid-content) {
+.content--immersive:not(.immersive-main-media__gallery):not(.paid-content):not(.content--gallery) {
```

Comment:
maybe we shouldn't really be hacking layouts on top of other layouts...? not a blocker for this though

Reply:
Yea it's definitely hacky, would require a bit of refactoring in this file though, should be separate.